sdk/avs/Azure.ResourceManager.Avs/src/Generated/Models/UnknownAvsPrivateCloudAddonProperties.cs

```diff
@@ -10,20 +10,20 @@
 
 namespace Azure.ResourceManager.Avs.Models
 {
-    /// <summary> Unknown version of AddonProperties. </summary>
-    internal partial class UnknownAddonProperties : AvsPrivateCloudAddonProperties
+    /// <summary> Unknown version of AvsPrivateCloudAddonProperties. </summary>
+    internal partial class UnknownAvsPrivateCloudAddonProperties : AvsPrivateCloudAddonProperties
```

Comment:
>  CannotChangeAttribute : Attribute 'System.ClientModel.Primitives.PersistableModelProxyAttribute' on 'Azure.ResourceManager.Avs.Models.AvsPrivateCloudAd
donProperties' changed from '[PersistableModelProxyAttribute(typeof(UnknownAddonProperties))]' in the contract to '[PersistableModelProxyAttribute(typeof(UnknownAvsPrivateCloudAddonProperties))]' in the implementation